### ISSUE TITLE
Creation of test data

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#36]
tasks for all team members

## Acceptance criteria
- [ ] data resembles the estimated style of email and ticket
- [ ] 40 per assigned person = 9 x 40 = 360

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by thogebati** [comment]
Example JSON:[https://github.com/amosproj/amos2023ws01-ticket-chat-ai/blob/7ddd4020574f1cb0ec33f23af5ad153a53e65455/Documentation/Ticket_Specifiacion/AMOS_Example_Ticket_PreIndustryPartner.json](url)

**Comment by M-HRL** [comment]
### Example JSON for our dataset

```json
 {
      "text": [
          "ABCDEF",
          "GHIJKL"
      ],
      "ticket": {
          "subject": "ABCDEFGHIJKLM",
          "description": "ABCDEFGHIJKLMNO",
          "impact": "low", // from ['low', 'meduim' 'high', 'very high']
          "priority": "low", // from ['low', 'meduim' 'high', 'very high']
          "request_type": "ABCDEF", // ['Service Request', 'Incidents', 'Problem'] this would be nice, but I can't give you a good explaniation what is what, so use this as reference "https://www.bmc.com/blogs/ticket-vs-incident-vs-problem-vs-service-request/"
      }
 }
```

**Comment by thogebati** [comment]
Both in 3 and 4
